### PR TITLE
Add description for Particles.capture_aabb and fix typo

### PR DESCRIPTION
**Pull request by Lisapple**
And fixing typo for `emission`.

### CODE REVIEW

**Comment by NathanLovato** [comment]
You opened 2 PRs with the same changes, closing this one. #29453

File: doc/classes/Particles.xml [hunk 1]
```diff
@@ -15,13 +15,14 @@
 			<return type="AABB">
 			</return>
 			<description>
+				Returns the AABB wrapping by all existing particles at the current frame.
```
Comment (NathanLovato) [inline doc/classes/Particles.xml]: Unclear, I think the grammar is off. 
Suggestion: Returns the AABB... bounding box? that encompasses/contains all the particles that are active in the current frame.

**Review by NathanLovato — CHANGES_REQUESTED** [review]
The grammar seems is off to me, and the description lacks precision. See my other comment for more info

File: doc/classes/Particles.xml (hunk 1 above)
Comment (Lisapple) [reply doc/classes/Particles.xml]: Yes, sorry about that. I've corrected the line.

**Review by NathanLovato — APPROVED** [review]

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.2.

**Pull request merged**